src/BaseProvider.js

```diff
@@ -66,31 +66,8 @@ const ChangeRenderer = (options) => {
 
 
 
-const CurrencyRenderer = ({unit, currency, decimals}) => {
-  unit = unit || 'mBTC';
-
-  const getFormat = (currency) => {
-    /* determined after mtgox api */
-    const front = "%s%v";
-    const back = "%v %s";
-    const frontFormats = {
-      USD: front, CAD: front, AUD: front, GBP: front,
-      HKD: front, NZD: front, SGD: front, THB: front,
-      MXN: front
-    };
-
-    return frontFormats[currency] || back;
-  };
-
-  const changeUnit = (number) => {
-    if (unit === 'mBTC') {
-      return Number(number) / 1000.0;
-    } else {
-      return Number(number);
-    }
-  };
-
-  let format = getFormat(currency);
+const CurrencyRenderer = ({currency, coin, decimals}) => {
```

Comment:
I think there is a way to implement your changes without sacrificing the old functionality. I'm not opposed to adding Altcoin support, but I don't want it to come at the cost of existing features. The mBTC support was specifically requested as a feature from some users.